packages/angular/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "@tanstack-query-firebase/angular",
+  "version": "0.0.1-alpha.5",
+  "description": "TanStack Query bindings for Firebase and Angular",
+  "type": "module",
+  "scripts": {
+    "copyToBuild": "node ./copyToBuild.js",
+    "build": "tsup && npm run copyToBuild",
+    "watch": "tsup --watch",
+    "test": "vitest --dom --coverage"
+  },
+  "typings": "./index.d.ts",
+  "module": "./index.js",
+  "exports": {
+    ".": {
+      "types": "./index.d.ts",
+      "import": "./index.js",
+      "default": "./data-connect/index.js"
+    },
+    "./data-connect": {
+      "import": "./data-connect/index.js",
+      "types": "./data-connect/index.d.ts",
+      "default": "./data-connect/index.js"
+    }
+  },
+  "author": "",
+  "license": "Apache-2.0",
+  "dependencies": {
+    "@tanstack/angular-query-experimental": "5.66.4"
+  },
+  "devDependencies": {
+    "@dataconnect/default-connector": "workspace:*",
+    "@analogjs/vite-plugin-angular": "^1.13.0",
+    "@angular/animations": "^19.0.0",
+    "@testing-library/angular": "^17.3.5",
+    "@testing-library/dom": "^10.4.0",
+    "tsup": "^8.4.0"
+  },
+  "peerDependencies": {
+    "@angular/core": "^19.0.0",
+    "@angular/fire": "^19.0.0",
+    "@angular/common": "^19.0.0",
+    "@angular/platform-browser-dynamic": "^19.0.0",
+    "firebase": "^11.1.0"
```

Comment:
just curious, is `firebase` really needed here? `@angular/fire` seems to avail all thats available through `firebase` 

aren't both offering the same things ?

Reply:
Removed peer dependency